### DISCUSSION TITLE
Consider documenting how bounties work

### DISCUSSION THREAD

**Discussion by matkoniecz** [maplibre/maplibre-gl-js#2221] [Q&A]
There are some issues marked with bounty labels but for example https://github.com/maplibre/maplibre-gl-js/issues?q=label%3A%22%F0%9F%92%B0+bounty+L%22+ have no info about what this bounty actually is, who is eligible, if it is payment - how large it is.

Maybe for some reason it is intentionally cryptic and hidden, but if not then mentioning in https://github.com/maplibre/maplibre-gl-js/blob/199be02a0c877516daf6e4e7cf3f4c46d11f0d48/CONTRIBUTING.md what is going on may be a good idea (maybe just link to https://github.com/maplibre/maplibre/wiki/Bounty-System )

https://github.com/sponsors/maplibre is nor mentioning bounties either

**Comment by HarelM** [comment]
Every bounty issue is linked to a bounty "direction", the link is inside the issue. I made sure they are there, it's part of the bounty requirements.
Every direction has a link to the bounty system, with the relevant amount and other information there in the maplibre wiki.
I'm not sure I know how to make it easier to understand.

**Reply by matkoniecz** [reply]
In my case I have seen this labels in general issue view. I tried clicking/hovering on label, looked in README, looked in CONTRIBUTING.MD file.

Now I see that I should:

- go to a specific issue like https://github.com/maplibre/maplibre-gl-js/issues/2194
- read comments
- go to https://github.com/maplibre/maplibre/issues/191
- follow "Read more about the approved Bounty Directions at https://maplibre.org/news/2023-01-20-bounty-directions/ "
- read until https://maplibre.org/news/2023-01-20-bounty-directions/#next-steps
- follow "The [#Bounty-System](https://github.com/maplibre/maplibre/wiki/Bounty-System) wiki page describes the full process."
- read https://github.com/maplibre/maplibre/wiki/Bounty-System

Not sure have I missed more obvious way to reach it.

(note: in my case I was specifically interested in how system is setup mostly because I am interested in how financing open source can work, not because I plan to apply for one of them)

**Comment by HarelM** [comment]
There's also this:
https://maplibre.org/roadmap/step-by-step-bounties-guide/

**Comment by wipfli** [comment]
Thanks for the feedback @matkoniecz. Would it help if we added some notes to the READMEs?

**Reply by matkoniecz** [reply]
yes (to avoid duplication link to the central documentation location on this topic would be probably the best)
